hal/armv7m/imxrt/117x/console.c

```diff
@@ -13,14 +13,27 @@
  * %LICENSE%
  */
 
+#ifndef UART_CONSOLE
+#define UART_CONSOLE 11
+#endif
+
 #include "cpu.h"
 #include "console.h"
 #include "imxrt117x.h"
 #include "../../include/errno.h"
 #include "../../include/arch/imxrt1170.h"
 
-#ifndef CONSOLE
-#define CONSOLE 11
+#define CONCAT(a, b)        a##b
+#define CONCAT2(a, b)       CONCAT(a, b)
+#define UART_PIN(n, pin)    CONCAT(UART, n##_##pin)
+#define CONSOLE_MUX(n, pin) (CONCAT2(pctl_mux_gpio_, UART_PIN(n, pin)))
+#define CONSOLE_PAD(n, pin) (CONCAT2(pctl_pad_gpio_, UART_PIN(n, pin)))
```

Comment:
Either use these in `info` or delete these

Reply:
I've removed unused macros.